src/main/java/org/jitsi/jigasi/TranscriptionGatewaySession.java

```diff
@@ -122,6 +122,16 @@ Exception onConferenceCallStarted(Call jvbConferenceCall)
         this.jvbCall = jvbConferenceCall;
         this.chatRoom = super.jvbConference.getJvbRoom();
 
+        // If the transcription service is not correctly configured, there is no
+        // point in continuing this session, so end it immediately
+        if(!service.isConfiguredProperly())
+        {
+            logger.warn("TranscriptionService is not properly set");
+            sendMessageToRoom("Transcriber is not properly configured. " +
+                    "Contact the host and let him now! I will now leave");
+            jvbConference.stop();
```

Comment:
Shouldn't we return here, before we add members and start the transcriber?

How about changing the chat message to something like "Contact the service administrators and let them know"?